src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -94,9 +95,22 @@ class ContainerBuilder extends Container implements TaggedContainerInterface
      */
     private $usedTags = array();
 
+    private $context;
+
     private $compiled = false;
 
     /**
+     * @param ParameterBagInterface|null $parameterBag A ParameterBagInterface instance
+     * @param Context|null               $context      A Context instance, or null
```

Comment:
Just as I did, you're missing this:

![capture d ecran 2016-08-18 a 16 46 00](https://cloud.githubusercontent.com/assets/2211145/17778123/4cdac800-6563-11e6-938c-3520a5ac8a9c.png)


😅